src/sandbox.ml

```diff
@@ -9,32 +9,38 @@ module Package = struct
   type t =
     | Opam of Opam.Package.t
     | Esy of Esy.Package.t
+    | Dune of Dune.Package.t
```

Comment:
Since we don't actually handle anything package related for dune (it's all failwith !), do we really have to add this case now ?